src/test/java/com/github/sergdelft/j2graph/representation/UnparsableCodeTest.java

```diff
@@ -0,0 +1,23 @@
+package com.github.sergdelft.j2graph.representation;
+
+import com.github.sergdelft.j2graph.TestBase;
+import com.github.sergdelft.j2graph.ast.JDT;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Tests that exercise the classes with imports.
+ * This was a bug (issue #3)
```

Comment:
is this really related to issue #3 ?

Reply:
Oh, that's a copy-paste error. Will fix.

Reply:
Done

Reply:
I still see 3 here?

Reply:
Oops, I fixed it in the Enum test. 🤦 Pushed for the unparsable code as well.

Reply:
Oops, I fixed the Enum test that had the same error. Added the commit for UnparsableCode. I guess the IllegalArgumentException is the expected output for unparsable code?